atomate/vasp/fireworks/core.py

```diff
@@ -84,9 +86,8 @@ def __init__(self, structure, name="structure optimization",
 
 
 class StaticFW(Firework):
-    def __init__(self, structure, name="static", vasp_input_set=None,
-                 vasp_cmd="vasp",
-                 prev_calc_loc=True, db_file=None, parents=None, **kwargs):
+    def __init__(self, structure, name="static", vasp_input_set=None, vasp_input_set_params=None,
```

Comment:
since you changed the order of the args (inserted a new arg in the middle) - have you confirmed that no existing code initializes this class without using keywords and assuming a particular argument order?

Probably not, but good to check at least ...